modules/common/src/main/resources/reference.conf

```diff
@@ -154,6 +154,7 @@ surge {
   }
 
   producer {
+    ask-timeout = 20 seconds
```

Comment:
Maybe `lifecycle-message-ask-timeout` or something? `ask-timeout` makes me think for all messages where this is really used for start/stop

Reply:
done

Reply:
Looks like this setting is unused now since swapping to `lifecycle-manager-actor.ask-timeout`